zenoh/path.py

```diff
@@ -18,8 +18,10 @@ def __init__(self, path):
         self.__path_regex = re.compile('^[^?#*]+$')
         if not self.is_valid(path):
             raise ValueError("Invalid Path: {}".format(path))
-        self.path = path
-
+        self._path = path
+    @property
```

Comment:
```suggestion

    @property
```
Add an empty line between the declarations for code readability.
Moreover, flake8 (linter for Python) is complaining about this missing empty line. See output of "make test" command.

Reply:
@Harmouch101 : Unfortunately this commit failed to pass the Eclipse ECA check, because you didn't sign it (-S option with git command). If you committed via the Github page, please check that you added your GPG key to your Github profile.

Reply:
@JEnoch ; ok, i will close this pull request and make another one with the right changes.